### PR TITLE
Add es translations

### PR DESCRIPTION
**Pull request by manuca**
Adds missing translations for the Spanish main locale (es):

- `es.spree.view_product`
- `es.spree.master_sku`
- `es.spree.admin.tabs.zones`
- `es.spree.filter`
- `es.spree.modify_stock_count`

### CODE REVIEW

**Comment by memotoro** [comment]
@jacobherrington Spanish looks ok in the file.

File: config/locales/es.yml [hunk 1]
```diff
@@ -894,7 +894,7 @@ es:
         users: Usuarios
         zones: Zonas
       taxons:
-        display_order: Mostrar Pedido
+        display_order: "Orden de presentación"
```
Comment (EduardoGHdez) [inline config/locales/es.yml]: 🤔 Not sure about this translation 👉 👈 

I preferred the other one.  But Spanish is so different depending of the region.

File: config/locales/es.yml [hunk 2]
```diff
@@ -871,7 +871,7 @@ es:
         areas: Ubicaciones
         checkout: Reembolsos y Devoluciones
         configuration: Configuración
-        display_order: Mostrar Pedido
+        display_order: Orden de presentación
```
Comment (EduardoGHdez) [inline config/locales/es.yml]: 🤔 Not sure about this translation 👉 👈

I preferred the other one. But Spanish is so different depending of the region.

Comment (manuca) [reply config/locales/es.yml]: Hi 👋 , the problem is that this is not related to Orders but to the order with which items are displayed on the screen. The previous translation was literally implying that this meant "displaying an order" which is wrong. 

Out of context is hard to spot this one, I noticed it on the Admin Panel, it simply didn't make sense.

Comment (memotoro) [reply config/locales/es.yml]: I agree with @manuca . I think that the translation makes sense as the menu makes reference to the order in which product within a taxon are displayed on listing pages and search.

Comment (EduardoGHdez) [reply config/locales/es.yml]: Gotcha!

In that case, totally make sense. Thank you so mucho for clear this doubt 🙌 

Thank you 😄

**Pull request merged**